projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/vdk_ui.py

```diff
@@ -152,22 +151,27 @@ def create_deployment(name: str, team: str, path: str, reason: str, enabled: boo
         :param enabled: flag whether the job is enabled (that will basically un-pause the job)
         :return: output string of the operation
         """
-        output = ""
+        output = "text"
         cmd = JobDeploy(RestApiUrlConfiguration.get_rest_api_url(), output)
-        path = get_or_prompt("Job Path", path)
```

Comment:
Why are we not prompting the user for these anymore?

Reply:
@DeltaMichael in the front end I already send him some messages about those, so doing it here is meaningless